pkg/placement/workload-projector.go

```diff
@@ -1104,7 +1104,7 @@ func (wp *workloadProjector) customizeOrCopy(logger klog.Logger, srcCluster logi
 }
 
 func kvIsSystem(which, key string) bool {
-	return (strings.Contains(key, ".kcp.io/") || strings.HasPrefix(key, "kcp.io/")) && !strings.Contains(key, "edge.kcp.io/")
+	return (strings.Contains(key, ".kcp.io/") || strings.HasPrefix(key, "kcp.io/")) && !strings.Contains(key, "edge.kubestellar.io/")
```

Comment:
@MikeSpreitzer this is because I did a bulk string replace. I didn't look at every line that was changed. I did something this for both edge.kcp.io and meta.kcp.io: 
grep -RiIl | xargs sed -i 's/meta.kcp.io/meta.kubestellar.io/g'
So there may be other instances like this that should be looked at.